pkg/kn/commands/service/service_update.go

```diff
@@ -36,8 +37,9 @@ func NewServiceUpdateCommand(p *commands.KnParams) *cobra.Command {
   kn service update mysvc --requests-cpu 500m --limits-memory 1024Mi`,
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
 			if len(args) != 1 {
-				return errors.New("requires the service name.")
+				return errors.New("'kn update' requires the service name")
```

Comment:
This isn't an accurate rendition of the actual command, and these changes seem unrelated to what you're doing in the rest of the PR. Can you make a separate PR?

Reply:
fair point, will do

Reply:
cleaned up and rebased